adapter/persistence/src/main/java/org/anastaklo/portal/adapter/persistence/RoomJpaEntity.java

```diff
@@ -0,0 +1,39 @@
+package org.anastaklo.portal.adapter.persistence;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+@Entity
+@Table(name = "room")
+public class RoomJpaEntity extends BaseJpaEntity {
+    
+    @Column
+    @Getter
+    @Setter
+    private String floor;
```

Comment:
string?

Reply:
I thought about something like '12A' but maybe it's an overthinking :) will change it to `int`